### ISSUE TITLE
Collapsed state vertical nav icons doesn't have textual representation

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4623]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:**  3.3.2 (Labels or Instructions)

**Test Environment:**
- Test URL: https://clarity.design/documentation/vertical-nav/basic-structure/project-pokemon
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL.
2. Go to the collapsible vertical nav which have icons, collapse that nav and check if user can see any tooltip for the focused clickable icon.

**Expected result:**
- In collapsed state if the vertical nav shows the visual icons of the nav menu items then the icons should have tooltip for describing actual menu item in a single focus i.e tooltip should be displayed when overall icon is focused, so that all users can understand the purpose of navigation menu items clearly.
- Also, ensure that visual tooltip matches the accessible name used in aria-label. Note: Tooltip should not be announced in expanded state to avoid redundant announcement of the menu item for screen reader users.
- Implemented tooltips should be available with mouse hover as well as with keyboard interaction. Keyboard event handlers needs to be added in addition to the mouseover event handlers which trigger the tooltips.

**Actual result:**
Tooltip is missing for visual icons. So users might not be able to use the collapsed state icons effectively.
In addition to that, Expand (>>) or Collapse (<<)  Toggle Navigation Menu button  visual tooltip is different than the accessible label.

**User Impact:**
https://www.w3.org/TR/WCAG20-TECHS/F82.html

**Comment by chlane** [comment]
This could also be fixed by using .clr-sr-only for the routing link text when the nav is collapsed rather than using display:none.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
